public_html/documentation.php

```diff
@@ -113,9 +174,29 @@ function build_sidebar_nav($elements) {
     $main_content .= '<div class="col-12 col-lg-2 order-lg-last ps-2 h-100 sticky-top"><div class="side-sub-subnav">';
     $main_content .= $toc_nav;
     $main_content .= '</div></div>'; # end of the sidebar col
-    # main content
-    $main_content .= '<div class="col-12 col-lg-8"><div class="rendered-markdown">' . $content . '</div></div>';
 
+    # main content
+    if($_GET['path'] === 'usage'){
+        $main_content .= '<div class="col-12 col-lg-3"><div class="rendered-markdown">';
+        foreach ($sidebar_nav_elements['usage'] as $mdfile => $mdcontent){
+            if (isset($mdcontent['url'])) {
+                $main_content .= '<div class="row mb-3 h3">' . '<a href="' . $mdcontent['url'] . '" class="text-success text-decoration-none">' . $mdcontent['title'] . '</div>';
+            }
+        }
+        $main_content .= '</div></div>';
+    }
+    if ($_GET['path'] === 'contributing'){
+        $main_content .= '<div class="col-12 col-lg-3"><div class="rendered-markdown">';
+        foreach ($sidebar_nav_elements['contributing'] as $mdfile => $mdcontent){
+          if (isset($mdcontent['url'])) {
+              $main_content .= '<div class="row mb-3 h3">' . '<a href="' . $mdcontent['url'] . '" class="text-success text-decoration-none">' . $mdcontent['title'] . '</div>';
+          }
+        }
+        $main_content .= '</div></div>';
+    }
```

Comment:
```suggestion
    # main content
    if(in_array($_GET['path'],['usage','contributing'])){
        $main_content .= '<div class="col-12 col-lg-3"><div class="rendered-markdown">';
        foreach ($sidebar_nav_elements[$_GET['path']] as $mdfile => $mdcontent){
            if (isset($mdcontent['url'])) {
                $main_content .= '<div class="row mb-3 h3">' . '<a href="' . $mdcontent['url'] . '" class="text-success text-decoration-none">' . $mdcontent['title'] . '</div>';
            }
        }
        $main_content .= '</div></div>';
    }
```
This is a bit more compact

Reply:
Nice! 👏